### PR TITLE
Twitter Mautrix from mattcen

### PR DESCRIPTION
**Pull request by shreyasajj**

### CODE REVIEW

**Comment by shreyasajj** [comment]
I think I fixed the one problem that he seems not have fixed

File: roles/matrix-bridge-mautrix-twitter/templates/config.yaml.j2 [hunk 1]
```diff
@@ -124,10 +124,10 @@ bridge:
         invite_own_puppet: true
         # Maximum number of messages to backfill initially.
         # Set to 0 to disable backfilling when creating portal.
-        initial_limit: 0
+        initial_limit: 100
```
Comment (spantaleev) [inline roles/matrix-bridge-mautrix-twitter/templates/config.yaml.j2:127]: Why `100`?

The default bridge config says `0`: https://github.com/mautrix/twitter/blob/e287a08b35fc36f03ecff4f5931b943644e76b6c/mautrix_twitter/example-config.yaml#L147

Comment (shreyasajj) [reply roles/matrix-bridge-mautrix-twitter/templates/config.yaml.j2:127]: Oh I forgot that I had an open pull request, I was just testing something on my local branch. I will roll it back for the last two  commitment. However it seems a change from `disable_notifications: true` on instagram for backfilling would be nice as a default. Would you like me to keep that in.

Comment (spantaleev) [reply roles/matrix-bridge-mautrix-twitter/templates/config.yaml.j2:127]: We usually try to stick to upstream defaults and only rarely deviate from them, so I guess it makes sense to restore to that.

**Comment by shreyasajj** [comment]
@spantaleev Fixed it

**Comment by spantaleev** [comment]
Thanks for getting this ready! :+1: 

I've updated the changelog to announce this new bridge.

**Comment by mattcen** [comment]
Thanks @shreyasajj and @spantaleev! 😁

**Comment by shreyasajj** [comment]
@mattcen you basically did 95% I did 5% just got it over the finish line

**Pull request merged**